packages/@coorpacademy-components/src/molecule/learning-priority-setup-item/index.js

```diff
@@ -0,0 +1,65 @@
+import React, {useCallback} from 'react';
+import PropTypes from 'prop-types';
+import Provider from '../../atom/provider';
+import Tag from '../../atom/tag';
+import ButtonLink from '../../atom/button-link';
+import style from './style.css';
+
+const LearningPrioritySetupItem = (props, context) => {
+  const {priorityRef, id, title, courses, type, 'aria-label': ariaLabel, onRemove} = props;
+
+  const {translate} = context;
+
+  const handleRemovePriority = useCallback(() => onRemove(priorityRef, type), [priorityRef, type]);
+
+  return (
+    <div
+      className={style.container}
+      aria-label={ariaLabel}
+      data-name={`learning-priority-setup-item-${id}`}
+    >
+      <div className={style.containerInfos}>
+        <div className={style.titleWrapper}>
+          <div className={style.title}>{title}</div>
+          <div>
+            <Tag label={translate(type)} size={'S'} />
```

Comment:
seem to have an issue with a test concerning the locales in this file,
maybe it's because of this, as is a variable and not a static string...
<img width="725" alt="image" src="https://github.com/user-attachments/assets/41f7bffc-d736-4cca-b066-f50625b4ae17">